.travis_build.sh

```diff
@@ -128,7 +128,7 @@ case "$PLATFORM" in
     pushd "${build_directory}"
     # remove bochs plugins
     sed -i 's/Bochs.* //g' plugins.ext
-    make
+	./mvm -f
```

Comment:
Hi, @nicolas-cellier-aka-nice 
this is not like that since a lot... my current version of this file reads like this: 

```bash -e ./mvm -f```

what are you checking?